SYCL/AtomicRef/add-native.cpp

```diff
@@ -0,0 +1,197 @@
+// RUN: %clangxx -fsycl -fsycl-unnamed-lambda -DSYCL_USE_NATIVE_FP_ATOMICS \
+// RUN: -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// TODO: Remove XFAIL/UNSUPPORTED once corresponding backends support
+// "native" SPIR-V
+// UNSUPPORTED: cuda
+// XFAIL: %CPU_RUN_PLACEHOLDER %t.out
```

Comment:
How do you think this works?
I think should be something like:
```suggestion
// RUN: %CPU_RUN_PLACEHOLDER %t.out
// XFAIL: cpu
```

Reply:
Done, sorry for missing stuff.